### PR TITLE
Let agent access rancher storage plugin sockets

### PR DESCRIPTION
**Pull request by imikushin**
Let agent access docker plugin sockets

### CODE REVIEW

**Comment by yasker** [comment]
LGTM.

It's needed because current rancher agent needs to talk with volume plugin to get volume attach/detached.

File: agent/run.sh [hunk 1]
```diff
@@ -164,6 +164,8 @@ launch_agent()
         -e CATTLE_MILLI_CPU_OVERRIDE \
         -e CATTLE_LOCAL_STORAGE_MB_OVERRIDE \
         -v /var/run/docker.sock:/var/run/docker.sock \
+        -v /run/docker/plugins:/run/docker/plugins \
```
Comment (yasker) [inline agent/run.sh]: Um... Why agent need to access `/run/docker/plugins` now?

Comment (imikushin) [inline agent/run.sh]: Ha... You're right! No need. Gotta remove it.

Comment (imikushin) [inline agent/run.sh]: Done

**Comment by yasker** [comment]
LGTM.

**Pull request merged**